feature/interface/aggregate/otg_tests/aggregate_all_not_viable_test/aggregate_all_not_forwarding_viable_test.go

```diff
@@ -0,0 +1,1025 @@
+		configForwardingViable(t, dut, dutPortList[agg2.ateLagCount:agg2.ateLagCount+1], true)
+		startTraffic(t, dut, ate, top)
+		if err := checkBidirectionalTraffic(t, dut, dutPortList[agg2.ateLagCount:agg2.ateLagCount+1]); err != nil {
+			t.Fatal(err)
+		}
+		if err := confirmNonViableForwardingTraffic(t, dut, ate, atePortList[1:agg2.ateLagCount], dutPortList[1:agg2.ateLagCount]); err != nil {
+			t.Fatal(err)
+		}
+		// Ensure there is no traffic received/transmiited on DUT LAG_3
+		if got := validateLag3Traffic(t, dut, ate, dutPortList[(agg2.ateLagCount+1):]); got == true {
+			t.Fatal("Packets are Received and Transmitted on LAG_3")
+		}
+		if ok := verifyTrafficFlow(t, ate, flows); !ok {
+			t.Fatal("Packet Dropped, LossPct for flow ")
+		}
+	})
+	// Reset Forwarding-Viable to True for all the ports of LAG_2
+	configForwardingViable(t, dut, dutPortList[1:6], true)
+	// Change ISIS metric Equal for Both LAG_2 and LAG_3
+	changeMetric(t, dut, aggIDs[2], 20)
+
+	t.Logf("ISIS cost of LAG_2 equal to ISIS cost of LAG_3 Test-02")
+	t.Run("RT-5.7.2.1: Setting Forwarding-Viable to False for Lag2 ports except port 2", func(t *testing.T) {
+		configForwardingViable(t, dut, dutPortList[2:agg2.ateLagCount+1], false)
+		flows = append(flows, configureFlows(t, top, pfx2AdvV4, pfx1AdvV4, "pfx2ToPfx1Lag3", agg3, []*aggPortData{agg1}))
+		ate.OTG().PushConfig(t, top)
+		ate.OTG().StartProtocols(t)
+		startTraffic(t, dut, ate, top)
+		if err := checkBidirectionalTraffic(t, dut, dutPortList[1:2]); err != nil {
+			t.Fatal(err)
+		}
+		if err := checkBidirectionalTraffic(t, dut, dutPortList[(agg2.ateLagCount+1):]); err != nil {
+			t.Fatal(err)
+		}
+		if err := confirmNonViableForwardingTraffic(t, dut, ate, atePortList[2:(agg2.ateLagCount+1)], dutPortList[2:(agg2.ateLagCount+1)]); err != nil {
+			t.Fatal(err)
+		}
+		// Ensure Load Balancing 50:50 on LAG_2 and LAG_3
+		weights := trafficRXWeights(t, ate, []string{agg2.ateAggName, agg3.ateAggName}, flows[0])
+		for idx, weight := range equalDistributionWeights {
+			if got, want := weights[idx], weight; got < want-ecmpTolerance || got > want+ecmpTolerance {
+				t.Errorf("ECMP Percentage for Aggregate Index: %d: got %d, want %d", idx+1, got, want)
+			}
+		}
+		if ok := verifyTrafficFlow(t, ate, flows); !ok {
+			t.Fatal("Packet Dropped, LossPct for flow ")
+		}
+	})
+
+	t.Run("RT-5.7.2.2: Setting Forwarding-Viable to False for Lag2 all ports", func(t *testing.T) {
+		// Ensure ISIS Adjacency is up on LAG_2
+		if ok := awaitAdjacency(t, dut, aggIDs[1], oc.Isis_IsisInterfaceAdjState_UP); !ok {
+			t.Fatal("ISIS Adjacency is Down on LAG_2")
+		}
+		configForwardingViable(t, dut, dutPortList[1:2], false)
+		// Ensure ISIS Adjacency is Down on LAG_2
+		if ok := awaitAdjacency(t, dut, aggIDs[1], oc.Isis_IsisInterfaceAdjState_DOWN); !ok {
+			t.Fatal("ISIS Adjacency is Established on LAG_2")
+		}
+		otgutils.WaitForARP(t, ate.OTG(), top, "IPv4")
```

Comment:
@rahulsy26  , I don't think we can expect to resolve ARP when all member of LAG are set to forwarding-viable  False. better add  dst mac address in all flows based on the destination port and remove wait for ARP for all cases. Other easy option could be add secondary mac mac address at system level in DUT and use  that MAC for all flows so that all ports can accept the traffic with that secondary mac. .  
 @dplore  could please suggest .

Reply:
Removed WaitForARP. Please suggest how to add dst mac address in all flows based on the destination port for reverse traffic (East to West direction)

Reply:
Remove WaitForArp() may resulted to traffic start as there is not destination MAC in flow.  we can add system level MAC and use that MAC for all flow  
gnmi.Replace(t, dut, gnmi.OC().System().MacAddress().RoutingMac().Config(), myStationMAC) [ https://github.com/openconfig/featureprofiles/blob/main/feature/experimental/interface/my_station_mac/otg_tests/my_station_mac_test/my_station_mac_test.go#L208] 

or other oprtion is used gnmi Get and get the DUT interface MAC and use that in respective flow .

Reply:
@bkreddy143 As you suggested, i tried above and with static_arp_test entry but looks like ixia is sending arp request to DUT even after i have added eV4.Dst().SetValue(dutAggMac).

https://github.com/openconfig/featureprofiles/blob/main/feature/interface/staticarp/otg_tests/static_arp_test/static_arp_test.go

Please let me know if any suggestion

Reply:
@rahulsy26 , from error message it looks like we might need to set static ARP in OTG  to pass protocol check , could you please check with IXIA/Keysight team  for the the same.  I did try to add gatewaymac but didnt work.

Reply:
Done